jobsdb/jobsdb.go

```diff
@@ -775,7 +775,7 @@ func (jd *Handle) init() {
 	}
 
 	if string(jd.conf.payloadColumnType) == "" {
-		jd.conf.payloadColumnType = payloadColumnType(jd.config.GetStringVar(string(JSONB), jd.configKeys("payloadColumnType")...))
+		jd.conf.payloadColumnType = payloadColumnType(jd.config.GetStringVar(string(TEXT), jd.configKeys("payloadColumnType")...))
```

Comment:
we may remove the configuration option altogether, but still support other data types in the rest of the code